setup.py

```diff
@@ -29,6 +29,7 @@ def read_file(filename, encoding='utf8'):
     'python-augeas',
     'python2-pythondialog',
     'requests',
+    'PyOpenSSL',
```

Comment:
unless there is a clear and documented reason (as for `M2Crypto`) this list should be sorted lexicogrpahically